src/utils/index.ts

```diff
@@ -4,3 +4,4 @@ export {KeyMap, KeyCode} from './key-map';
 export {default as getLogger} from './logger';
 export {withKeyboardA11y} from './popup-keyboard-accessibility';
 export * from './time-format';
+export {getTimeInText} from './duration-humanizer';
```

Comment:
I think we should be very clear in the name of the method so it would be intuitive what it does.

Time => i think it should be "duration". because we're not handling timestamps, just duration in seconds.
InText => I think it should be "as text"

So I would prefer something like "getDurationAsText" more.

Reply:
done